uint32 out_class
float32 out_score
float32[] location